### PR TITLE
Integrate build_US_setup into pipeline and...

### LINKED ISSUE #248: Update build us setup to use tidycensus

### PR DESCRIPTION
**Pull request by kkintaro**
pull population data using tidycensus, except for island areas: American Samoa, Guam, Virgin Islands, and Mariana Islands.

Closes #248  

Okay to squash, merge, delete

### CODE REVIEW

File: R/scripts/build_US_setup.R [hunk 1]
```diff
@@ -9,49 +42,96 @@ option_list = list(
...
+if (length(config$spatial_setup$geodata) > 0) {
+  geodata_file <- config$spatial_setup$geodata
+} else {
+  geodata_file <- 'geodata.csv'
+}
+write.csv(file = file.path(outdir, geodata_file), census_data, row.names=FALSE)
+print(paste("Wrote geodata file:", file.path(outdir, geodata_file)))
+
+# COMMUTE DATA
+commute_data <- readr::read_csv(paste(opt$p,"sample_data","united-states-commutes","commute_data.csv",sep='/'))
```
Comment (eclee25) [inline R/scripts/build_US_setup.R:123]: just checking: does opt$p need to be defined above?

File: R/scripts/build_US_setup.R [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+#
+
+## @cond
+
 library(dplyr)
 library(tidyr)
+library(tidycensus)
 
 option_list = list(
   optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("CONFIG_PATH"), type='character', help="path to the config file"),
```
Comment (jkamins7) [inline R/scripts/build_US_setup.R:38]: @eclee25 it's defined here

File: R/scripts/build_US_setup.R (hunk 1 above)
Comment (kkintaro) [reply R/scripts/build_US_setup.R:123]: The default is "COVIDScenarioPipeline" if it's not provided by user on the command line. Should be line 39.

File: R/scripts/build_US_setup.R (hunk 2 above)
Comment (jkamins7) [reply R/scripts/build_US_setup.R:38]: well, one line below the last one visible

File: R/scripts/build_US_setup.R [hunk 3]
```diff
@@ -62,20 +142,31 @@ t_commute_table <- tibble(
 )
 
 rc <- padding_table %>% bind_rows(commute_data) %>% bind_rows(t_commute_table)
+
 if(opt$w){
-  rc <- rc %>%pivot_wider(OFIPS,names_from=DFIPS,values_from=FLOW, values_fill=c("FLOW"=0),values_fn = list(FLOW=sum))
+  mobility_file <- 'mobility.txt'
+} else if (length(config$spatial_setup$mobility) > 0) {
```
Comment (eclee25) [inline R/scripts/build_US_setup.R:148]: So the behavior here will be that the config filename (if it ends with .txt) overwrites the -w option in the config call

Comment (eclee25) [reply R/scripts/build_US_setup.R:148]: I guess this will be good for backward compatibility

**Pull request merged**